cli/command/image/trust.go

```diff
@@ -294,7 +294,12 @@ func trustedPull(ctx context.Context, cli *command.DockerCli, repoInfo *registry
 		if err != nil {
 			return err
 		}
-		if err := imagePullPrivileged(ctx, cli, authConfig, ref.String(), requestPrivilege, false); err != nil {
+
+		// TODO(icecrime): Should we allow per-pull insecure registries overrides for trusted pull
+		// operations? It seems contradictory, and would possibly even be better to error out when
+		// `--insecure-registry` is specified for a pull operation with Docker Content Trust
```

Comment:
With DCT enabled, wouldn't `insecure-registry` be more secure, as we can verify the incoming content against a central authority?